t/Makefile.am

```diff
@@ -80,7 +80,8 @@ test_meta_cppflags = \
 	-I$(top_srcdir)/common/src \
 	-D_GNU_SOURCE \
 	$(AM_CPPFLAGS) \
-        $(MPI_CFLAGS)
+    $(MARGO_CFLAGS) \
```

Comment:
Spacing seems to be off a bit, spaces vs tabs?

Reply:
@adammoody I think original one is with tab, while my vim puts four spaces instead. I will change to use the tab, although we currently mix the use of tabs and spaces.